spec/spec_helper.rb

```diff
@@ -47,7 +48,7 @@ def fixture(*name)
 # of the letters and numbers work.
 def ascii(from = 33, to = 126)
   chars = []
-  from.upto(to) { |c| chars << ('' << c) }
+  from.upto(to) { |c| chars << (String.new('') << c) }
```

Comment:
ok -- but not sure this change really has anything to do with a PR thats about frozen string literals